### ISSUE TITLE
User management for self hosted instances

### ISSUE THREAD

**Issue by Luxizzle** [texterify/texterify#58]
I'd like to manage users in my self hosted instance globally, not just per organization. Even better if i can disable account creation completely and create accounts for new users manually. Currently we can only see the amount of users in an instance, but we have no control over who can create an account, only solution now is to keep it behind a strict firewall or VPN.

**Comment by chrztoph** [comment]
Good point. That's something to add as soon as possible. I will add an option to disable account creation next week and also the possibility to specify a filter that only users with an email address of a given domain can sign up. Also the possibility to add users manually is a good idea.

Thanks for your feedback. I will keep you updated here. 🙂

**Comment by chrztoph** [comment]
If you upgrade to version `chrztoph/texterify:v1.0.1` there is now an option in the admin interface to disable sign up. Only users who have a pending invite are then allowed to create an account so you can control who gets access to your instance.

Global user management will be added at a later point.
